plugin-server/tests/server.test.ts

```diff
@@ -10,7 +10,7 @@ import { resetTestDatabase } from './helpers/sql'
 jest.mock('../src/utils/kill')
 jest.mock('../src/main/graphile-worker/schedule')
 jest.mock('../src/main/graphile-worker/worker-setup')
-jest.setTimeout(20000) // 20 sec timeout - longer indicates an issue
+jest.setTimeout(30000) // 30 sec timeout - longer indicates an issue
```

Comment:
@benjackwhite you added this comment according to git blame... I'm aiming to hand-wave that another service means it takes slightly longer to stop and ignore bumping this timeout 🙈

Reply:
I think my comment was basically "If the test passes, it passes fast - longer than a few seconds generally indicates that something actually went wrong and waiting longer doesn't help".

That may not be true of the server.test.ts though...

Reply:
seems to stop them failing so let's 🚢 and 🙈